change/@microsoft-teams-js-2b4256da-5891-42da-b857-e298ef5d7dac.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Move userJoinedTeams",
```

Comment:
Let's include more details in the changelog, maybe something like "legacy.fullTrust.getUserJoinedTeams() has been moved into its own subcapability called joinedTeams and is now legacy.fullTrust.joinedTeams.getUserJoinedTeams()."